src/main/java/com/palantir/ptoss/cinch/core/BindingContext.java

```diff
@@ -192,6 +192,21 @@ public Object evalOnObject(String on, BindableModel model) {
         return findOnObject(on, model);
     }
 
+    public static boolean isNullOrBlank(String string) {
+        if (string == null) {
+            return true;
+        }
+        int length = string.length();
+        if (length > 0) {
+            for (int i = 0; i < length; i++) {
+                if (!Character.isWhitespace(string.charAt(i))) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
```

Comment:
Could we move this to a util class that's clearly marked as non-public api?

Reply:
made this instance private, duplicated to package visible `com.palantir.ptoss.cinch.swing.Bound.Utilities#isNullOrBlank`